src/app/utils/getNews.ts

```diff
@@ -4,7 +4,7 @@ import axios from 'axios';
 import { z } from 'zod';
 import objectToQueryString from './objectToQueryString';
 
-const getNews = async (keyword: string, params: Selects = {}): Promise<NewsData> => {
+const getNews = async (keyword: string, page: string, params: Selects = {}): Promise<NewsData> => {
```

Comment:
`page` is unused

Reply:
change logic with page number in url, now it's used)